src/bgzip/config.vsh.yaml

```diff
@@ -0,0 +1,45 @@
+functionality:
+  name: bgzip
+  description: Block compression/decompression utility
+  info:
+    homepage: https://www.htslib.org/
+    documentation: https://www.htslib.org/doc/bgzip.html
+    repository: https://github.com/samtools/htslib
+    licence: MIT
+  requirements:
+    cpus: 1
+    commands: [ bgzip ]
+  arguments:
+    - name: --input
+      type: file
+      direction: input
+      description: file to be compressed or decompressed
+      required: true
+    - name: --output
+      type: file
+      direction: output
+      description: compressed or decompressed output
+      required: true
+    - name: --extra
+      type: string
+      description: extra arguments to pass to bgzip
+      required: false
+    - name: --threads
+      type: integer
+      description: number of threads to use
+      required: false
+      default: 1
```

Comment:
Don't create a separate argument for this, instead use `$meta_cpus` in your script

Reply:
Fixed!